### PR TITLE
feat: pass callback parameters

### PR DESCRIPTION
**Pull request by flappyBug**
I want to get position when onResizeEnd. This is useful to record the position in local storage.

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -67,11 +67,11 @@ export type UseResizableProps = {
   /**
    * callback when border position changes start
    */
-  onResizeStart?: () => void;
+  onResizeStart?: (position: number) => void;
```
Comment (RyoSogawa) [inline src/types.ts:70]: @flappyBug 
Thanks

Comment (RyoSogawa) [inline src/types.ts:70]: @flappyBug 
Thank you for the intriguing feature❤️ 

One suggestion: considering the potential addition of parameters in the future, how about making the arguments into an object?

```ts
  export type ResizeCallbackArgs = {
    position: number;
  }
  // ...
  onResizeStart?: (args: ResizeCallbackArgs) => void;
  onResizeEnd?: (args: ResizeCallbackArgs) => void;
```

Comment (flappyBug) [reply src/types.ts]: Sure. Thanks for the suggestion. Updated

**Review by RyoSogawa — APPROVED** [review]
@flappyBug 
Thanks for your contribution!
We will be releasing a new version soon.

**Pull request merged**